vendor/assets/stylesheets/ustyle/components/_hero.scss

```diff
@@ -44,6 +48,8 @@ $hero-text-color: $c-navy !default;
 .us-hero-description {
   @extend %reset-box-model;
   display: none;
+  padding-top: 5px;
+  color: $c-typegrey;
```

Comment:
What colour is it currently?

Reply:
Depends where you look. On Bad Credit (And other places I imagine), it's inheriting the body which is c-navy (#1c1f4e) and others use c-typegrey (#2f353a). I used the colour that was on Bad Credit.